aries_cloudagent/askar/tests/test_profile.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from asynctest import mock as async_mock
+
+from asynctest import TestCase as AsyncTestCase
+
+from .. import profile as test_module
+from ...core.profile import Profile, ProfileSession
+from ...core.in_memory import InMemoryProfile
+
+class testProfile(AsyncTestCase):
+  @pytest.mark.asyncio
+  async def test_profile_manager_transaction(self):
+
+    with async_mock.patch(
+    "aries_cloudagent.askar.profile.AskarProfile"
+    ) as AskarProfile:
+      askar_profile = AskarProfile(None, True)
+      askar_profile_return = async_mock.MagicMock()
+      askar_profile.store.transaction.return_value = askar_profile_return
+      askar_profile.context.settings.get.return_value = "walletId"
```

Comment:
Function that would generate askar_profile could be created in order to avoid duplication between tests